common/Services/IPluginWrapper.cs

```diff
@@ -23,6 +25,22 @@ string PackageFullName
         get;
     }
 
+    /// <summary>
+    /// Gets PackageFamilyName of the extension
+    /// </summary>
+    string PackageFamilyName
+    {
+        get;
+    }
+
+    /// <summary>
+    /// Gets Publisher of the extension
+    /// </summary>
+    string Publisher
+    {
+        get;
+    }
+

```

Comment:
(discussed below) Add property here for the Unique Identifier of the extension (Aumid + ExtensionId).